src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -468,7 +468,7 @@ function ActionButtons({ earnPosition }: { earnPosition: EarnPosition }) {
               networkId: earnPosition.networkId,
               depositTokenId: earnPosition.dataProps.depositTokenId,
             })
-            navigate(Screens.EarnEnterAmount, { pool: earnPosition })
```

Comment:
@satish-ravi was this meant to be included?

Reply:
oops, good catch